core/src/main/java/org/apache/iceberg/rest/responses/ListNamespacesResponse.java

```diff
@@ -59,6 +68,7 @@ public static Builder builder() {
 
   public static class Builder {
     private final ImmutableList.Builder<Namespace> namespaces = ImmutableList.builder();
+    private String nextPageToken = null;
```

Comment:
this would require a builder method so that the value can be set

Reply:
will add